ReactQt/runtime/src/reactnavigatormanager.cpp

```diff
@@ -85,89 +85,96 @@ void UbuntuNavigatorManager::clear(int containerTag)
 }
 
 
-UbuntuNavigatorManager::UbuntuNavigatorManager(QObject* parent)
+ReactNavigatorManager::ReactNavigatorManager(QObject* parent)
   : ReactViewManager(parent)
   , m_id(0)
 {
 }
 
-UbuntuNavigatorManager::~UbuntuNavigatorManager()
+ReactNavigatorManager::~ReactNavigatorManager()
 {
 }
 
-void UbuntuNavigatorManager::setBridge(ReactBridge* bridge)
+void ReactNavigatorManager::setBridge(ReactBridge* bridge)
 {
   m_bridge = bridge;
 }
 
-ReactViewManager* UbuntuNavigatorManager::viewManager()
+ReactViewManager* ReactNavigatorManager::viewManager()
 {
   return this;
 }
 
-ReactPropertyHandler* UbuntuNavigatorManager::propertyHandler(QObject* object)
+ReactPropertyHandler* ReactNavigatorManager::propertyHandler(QObject* object)
 {
   Q_ASSERT(qobject_cast<QQuickItem*>(object) != nullptr);
   return new NavigatorPropertyHandler(object);
 }
 
-QString UbuntuNavigatorManager::moduleName()
+QString ReactNavigatorManager::moduleName()
 {
-  return "UbuntuNavigatorManager";
+  return "RCTNavigatorManager";
```

Comment:
It's might be better to start naming of modules without "RCT" prefix.
"RCT" prefix is some kind of old naming standard for react-native modules.
https://github.com/status-im/react-native-linux/blob/react-native-qt/ReactQt/runtime/src/reactmoduledata.cpp#L116